src/transformers/models/oneformer/modeling_oneformer.py

```diff
@@ -625,7 +625,10 @@ def sample_points_using_uncertainty(
 
         if num_random_points > 0:
             point_coordinates = torch.cat(
-                [point_coordinates, torch.rand(num_boxes, num_random_points, 2, device=logits.device)],
+                [
+                    point_coordinates,
+                    torch.rand(num_boxes, num_random_points, 2, device=logits.device),
+                ],
```

Comment:
Let's keep this on one line 

```suggestion
                [point_coordinates, torch.rand(num_boxes, num_random_points, 2, device=logits.device)],
```